plasma_framework/docs/design/ald-exit-game-definitions-and-properties.md

```diff
@@ -0,0 +1,133 @@
+# ALD Exit Game Definitions And Properties
+
+## New definitions from ALD (abstract layer design)
+Based on [MoreVP document](https://github.com/omisego/elixir-omg/blob/master/docs/morevp.md). This section adds some new definitions introduced from ALD or give explicit names to some of the implicit characteristics of the origin MoreVP spec.
+
+### SpendingCondition Fulfilled
+1. Checks an input tx and the spending tx, whether the spending tx fulfil the requirement of spending the output of input tx or not.
+1. Basic checks:
+    - Spending tx input points to the output of input tx.
+    - Spending tx fulfil the “lock” condition of the output
+1. This checks on (at most) tx level, does not check things on Plasma protocol level.
+
+### Tx Standard Finalization
+1. MVP:
+    - Tx is in a plasma block. (Checks inclusion proof of plasma block)
+    - It is confirmed by user. (Checks confirm signature is valid for the inclusion block)
+2. MoreVP:
+    - Tx is in a plasma block. (Checks inclusion proof of plasma block)
+
+### Tx Protocol Finalized
+This is the minimal finalization requirement according to each protocol.
+
+1. MoreVp tx: the transaction exists
```

Comment:
What does that mean?

Reply:
ehh...literally as there is/exists a transaction.

Reply:
I guess I am trying to understand what Protocol Finalized means. So in MoreVP it means any tx is protocol finalized there are no special requirements on the protocol level. For MVP it states that it is standard finalized so referring to the this section https://github.com/omisego/plasma-contracts/pull/455/files#diff-c977e911b5a05982bacd0a92e035dbd3R13?